build.gradle

```diff
@@ -55,7 +55,7 @@ repositories {
 
 }
 
-final htsjdkVersion = System.getProperty('htsjdk.version','2.8.1-22-gb05d1f2-SNAPSHOT')
+final htsjdkVersion = System.getProperty('htsjdk.version','2.9.0-1-g55bf01b-20170217.164935-1')
```

Comment:
There can be more than one snapshot for a given commit hash -- by including the timestamp, we can select a specific snapshot. Not strictly necessary, but I think it's nice having the snapshot date right in the version number, for informational purposes.